src/check_input.f90

```diff
@@ -369,7 +371,7 @@ subroutine check_ccmc_opts(sys, ccmc_in)
         end if
 
         if (ccmc_in%density_matrices .and. sys%system /= read_in) then
-            call stop_all(this, "CCMC density matrices not implemented for this system type.")
+            call stop_all(this, "2-body density matrices not implemented for this system type.")
```

Comment:
Why did you change CCMC to 2-body? I have a felling ccmc density matrices might include a form of the 1RDM? @verena-neufeld ?